user-creation/templates/user-manifests.yaml

```diff
@@ -29,13 +29,30 @@ metadata:
     {{ $labels }}
 spec:
   limits:
+  # default limits per Pod
+  # using 1:4 ratio as default
   - default:
+      cpu: 125m
       memory: 512Mi
     defaultRequest:
       cpu: 10m
       memory: 32Mi
     type: Container
 
+---
+# Resource Quotas per user, e.g. namespace
+# https://kubernetes.io/docs/tasks/administer-cluster/quota-api-object/
```

Comment:
documentation: request can never be greater than the limit => a malicous user can not set the request to e.g. 15GB while conforming to a set limit of 2GB